scripts/install/setup_properties.sh

```diff
@@ -46,9 +46,10 @@ else
       SERVICE_ACCOUNT_NAME=$(echo $EXISTING_SA_EMAIL | cut -d @ -f 1)
     fi
   fi
-
+  NETWORK="default"
   ZONE=${ZONE:-us-west1-b}
   REGION=$(echo $ZONE | cut -d - -f 1,2)
+  SUBNET=$(gcloud compute networks subnets list --network=default --filter="region:( $REGION )" | cut -d " " -f 1 |sed -n 2p)
```

Comment:
If we use a `gcloud compute` command here, it will on a new project prompt for user input and then pause for several minutes while the Compute api is enabled.

This script is meant to complete quickly and to not require user input (it is sometimes used to provision infrastructure for integration tests).

Can we just start with `default` like we do with `NETWORK`? Seems just as valid as a default value. Could still verify the subnet exists in `setup.sh` like we do with `NETWORK`.

Reply:
And would move the `SUBNET=` assignment to just below the `NETWORK=` assignment.

Reply:
would it make sense to set it as default, and then allow for logic within the setup.sh to verify that the subnet exists and if not run this to pull the subnet name?

Reply:
I like that idea in principle, only issue I see is that what's captured in the properties file may end up not matching the actual subnet that was resolved.

Reply:
what if we just check in the setup.sh script that this exists, and if not error out.

Reply:
That sounds good to me.

Reply:
I have made these updates to both of the files